app/views/application/_object_summary.html.erb

```diff
@@ -14,11 +14,9 @@
       <%= current_object.class.to_s %> <%= current_object.pid %><br/>
       <hr/>
     </li>
-    <% if current_object.respond_to?(:identifier) && current_object.identifier.present? %>
-      <li class="small">
-        <%= current_object.identifier.first %>
-      </li>
-    <% end %>
+    <li class="small">
+      <%= current_document.local_id %>
+    </li>
```

Comment:
You want an empty `<li>` when local_id is not present?

Reply:
@dchandekstark Don't really care ... that is current behavior if there isn't an identifier present.